types/react/index.d.ts

```diff
@@ -26,6 +26,7 @@ declare namespace React {
     // ----------------------------------------------------------------------
 
     type ReactType = string | ComponentClass<any> | StatelessComponent<any>;
```

Comment:
sorry, just one more thing—let's change this to `string | ComponentType<{}>`. otherwise looks good!